### PR TITLE
DOC: SS01 docstrings errors fixed

### PR DESCRIPTION
**Pull request by lucasrodes**
xref #27977
- [ ] closes part of [MarcoGorelli/PyDataGlobal2020-sprint#4](https://github.com/MarcoGorelli/PyDataGlobal2020-sprint/issues/4)
- [ ] tests added / passed
- [X] passes `black pandas`
- [ ] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

### CODE REVIEW

File: pandas/_libs/lib.pyx [hunk 1]
```diff
@@ -927,6 +929,8 @@ def indices_fast(ndarray index, const int64_t[:] labels, list keys,
 
 def is_float(obj: object) -> bool:
     """
+    Return a boolean if this is a float object.
```
Comment (MarcoGorelli) [inline pandas/_libs/lib.pyx:932]: I think these are the same as above (`Return True`)

**Comment by MarcoGorelli** [comment]
Does this fix all the SS01 errors? If so, do you want to add this error code to the relevant check in `scripts/code_checks.sh`?

**Comment by lucasrodes** [comment]
Sure @MarcoGorelli , thanks

File: pandas/_libs/lib.pyx (hunk 1 above)
Comment (lucasrodes) [reply pandas/_libs/lib.pyx:932]: changing docstrings to 

```
Return True if given object is [type].
```

**Comment by MarcoGorelli** [comment]
I think you accidentally committed path_to_file.zip ?

**Comment by lucasrodes** [comment]
@MarcoGorelli Indeed... Got it removed!

**Review by jorisvandenbossche — APPROVED** [review]
Looks good, thanks!

**Comment by MarcoGorelli** [comment]
Thanks @lucasrodes , and congrats on your first PR to pandas!

**Pull request merged**